### DISCUSSION TITLE
A "Settings Override" for infill pattern would be very useful.

### DISCUSSION THREAD

**Discussion by moccor** [OrcaSlicer/OrcaSlicer#6897] [Ideas]
Cura has many settings like this (not sure if I used a plugin to get them). But basically, some filament materials do better with different infill. For example, TPU "squish" (strength) greatly varies depending on infill. It would be very convenient to be able to just switch the filament and then not have to think about infill types for things like that. TPU does very good with gyroid for "squish", but the reason I wouldn't want it left enable is because gyroid also greatly increases print times to me. Another example would be for see-through or Glow in the Dark filaments, infill patterns that touch the wall often can be visible unless adding a few extra walls, which adds a lot of print time.

Honestly, a "override" for all of the slicer settings would be very nice to have. Another example is, TPU not needing a brim whatsoever, while in colder months PLA, PETG and ABS would benefit greatly from having a brim. A override to just have brims disabled for TPU would be nice

**Comment by alaorath** [comment]
Agreed.  For PETG, I use a "non crossing" infill pattern, but PLA is fine with anything.  So I have a 'process profile' for PETG to set to cross hatch.

Would be nice to have this part of the filament profile instead of setting up Process Profiles per filament.
